File: benchmark/compare/standard/function_call.py
import time

def add(a, b):
    return a + b

start = time.perf_counter()
result = 0
for i in range(100000):
    result = add(result, i)
elapsed = (time.perf_counter() - start) * 1000
print(f"result = {result}")
print(f"elapsed_ms={elapsed:.2f}")

File: benchmark/compare/standard/function_call.js
function add(a, b) { return a + b; }
let start = performance.now();
let result = 0;
for (let i = 0; i < 100000; i++) { result = add(result, i); }
let elapsed = performance.now() - start;
console.log(`result = ${result}`);
console.log(`elapsed_ms=${elapsed.toFixed(2)}`);
